### ISSUE TITLE
make it possible to hide code completely in standalone

### ISSUE THREAD

**Issue by dktr0** [dktr0/Punctual#14]

**Comment by totalgee** [comment]
Yes, this would be great, to be able to temporarily toggle it off.

Also, or as a workaround, it would be nice to be able to scroll the code with the mouse wheel (trackpad) so you could include a bunch of blank lines to "clear the screen" and then quickly scroll back to the code when you want to continue. Right now you need to move the cursor (arrow keys or PgUp/PgDn) to do that, but then you lose your place!
